README.md

```diff
@@ -108,7 +109,8 @@ engine = create_engine(
     'trino://user@localhost:8080/system?'
     'session_properties={"query_max_run_time": "1d"}'
     '&client_tags=["tag1", "tag2"]'
-    '&experimental_python_types=true',
+    '&experimental_python_types=true'
+    '&roles={"catalog1": "role1"}'
```

Comment:
no test for sqlalchemy integration?

Reply:
See https://github.com/trinodb/trino-python-client/pull/212#discussion_r978841837